### PR TITLE
feat: OPTIC-107: Draft saving on view all

### PR DESCRIPTION
**Pull request by Travis1282**
### PR fulfills these requirements
- [X] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [X] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [X] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [X] Frontend

### Describe the reason for change
drafts were being lost when view all was selected, this change ensures they are saved and notifies the user accordingly. 

#### What does this fix?
_(if this is a bug fix)_

#### What is the new behavior?
_(if this is a breaking or feature change)_

#### What is the current behavior?
_(if this is a breaking or feature change)_

#### What libraries were added/updated?
_(list all with version changes)_

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_

#### Does this change affect security?
_(if so describe the impacts positive or negative)_

#### What alternative approaches were there?
_(briefly list any if applicable)_

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: src/core/TimeTraveller.js [hunk 1]
```diff
@@ -77,6 +77,7 @@ const TimeTraveller = types
       },
 
       onUpdate(handler) {
+        this.lastAdditionTime = new Date();
```
Comment (yyassi-heartex) [inline src/core/TimeTraveller.js:80]: does the lastAdditionTime get saved to the annotation? if so there could be issues around users from differnt time zones interacting with it.
aside from that the concept makes sense and im ready to approve - just wondering about this

**Review by yyassi-heartex — APPROVED** [review]

File: src/core/TimeTraveller.js (hunk 1 above)
Comment (Travis1282) [reply src/core/TimeTraveller.js:80]: it does not,  good thinking tho

**Pull request merged**